Source/WebCore/html/HTMLDetailsElement.cpp

```diff
@@ -159,6 +159,15 @@ void HTMLDetailsElement::attributeChanged(const QualifiedName& name, const AtomS
             if (m_isOpen) {
                 root->appendChild(*m_defaultSlot);
                 queueDetailsToggleEventTask(DetailsState::Closed, DetailsState::Open);
+                if (auto& name = attributeWithoutSynchronization(nameAttr); !name.isEmpty()) {
```

Comment:
Can we gate this code behind a preference (enabled by default is fine, I just want to make sure we have an off switch for the feature if needed)? See https://searchfox.org/wubkat/search?q=inertAttributeEnabled&path=Source&case=false&regexp=false for instance.